brave/brave-core
# Adds correct captcha image

Resolves brave/brave-browser#928

## Submitter Checklist:


## Test Plan:

1. clean profile
2. enable rewards
3. click claim grant
4. make sure that you see image with 3 coloured triangles
5. make sure that you can solve captcha

## Reviewer Checklist:
